src/coreComponents/linearAlgebra/interfaces/hypre/mgrStrategies/SolidMechanicsEmbeddedFractures.hpp

```diff
@@ -64,7 +64,7 @@ class SolidMechanicsEmbeddedFractures : public MGRStrategyBase< 1 >
     setupLabels();
 
     // Level 0
-    m_levelFRelaxType[0]         = MGRFRelaxationType::gsElimWInverse;
+    m_levelFRelaxType[0]         = MGRFRelaxationType::jacobi;
```

Comment:
what is the effect of this?

Reply:
My guess is that the `gsElimWInverse` is not really scalable. Shouldn't we do block jacobi?

Reply:
Yes, we should be doing block jacobi

Reply:
Sorry, now I recall what's going on here...

Under the hood, hypre does block-Jacobi because of the interpolation choice `m_levelInterpType[0] = MGRInterpolationType::blockJacobi`. However, this logic is confusing (it shouldn't depend on the interpolation) and hypre should have a specific option for doing block-Jacobi F-relaxation.

I can work on that.